packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol

```diff
@@ -308,24 +311,36 @@ abstract contract CrossDomainMessenger is
         }
     }
 
+    /// @notice
+    function getCrossDomainMessageSender() internal view returns (address) {
```

Comment:
this could also be transient storage

Reply:
We unfortunately won't have the solidity version update in yet.